src/lava/magma/runtime/runtime_service.py

```diff
@@ -113,13 +117,12 @@ def _send_pm_req_given_model_id(self, model_id: int, *requests):
         for request in requests:
             req_port.send(request)
 
-    def _get_pm_resp(self, phase) -> ty.Iterable[MGMT_RESPONSE]:
+    def _get_pm_resp(self) -> ty.Iterable[MGMT_RESPONSE]:
```

Comment:
Please add docstrings and inline explanations.

Reply:
Done